src/test/scala/org/zalando/spearheads/innkeeper/services/RouteToEskipTransformerSpec.scala

```diff
@@ -41,6 +41,20 @@ class RouteToEskipTransformerSpec extends FunSpec with Matchers with MockFactory
       routeToEskipTransformer.transform(contextWithRouteWithoutEndpoint) should
         be(expectedResult.copy(endpoint = "<shunt>"))
     }
+
+    it("should transform a route which has a star path to an eskip route") {
```

Comment:
We'll keep only one route as the output. The solution for this issue will be solved in skipper + innkeeper.